components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2LoginEventPayloadBuilder.java

```diff
@@ -146,14 +146,14 @@ private List<String> buildAuthMethods(AuthenticationContext authContext) {
     private Reason buildAuthenticationFailedReason(AuthenticationContext authContext) {
 
         HashMap<String, String> dataMap = (HashMap<String, String>) authContext.getParameters().get(Constants.DATA_MAP);
-        String errorCode = dataMap.get(Constants.CURRENT_AUTHENTICATOR_ERROR_CODE);
+//        String errorCode = dataMap.get(Constants.CURRENT_AUTHENTICATOR_ERROR_CODE);
```

Comment:
remove this we should have this error code

Reply:
Fixed 371e89b5873a1f496a54ed4a843e2838a694fcd9